src/exchange_adapters/binance_us.ts

```diff
@@ -5,7 +5,7 @@ import { BinanceAdapter } from './binance'
 export class BinanceUSAdapter extends BinanceAdapter implements ExchangeAdapter {
   baseApiUrl = 'https://api.binance.us/api/v3'
   readonly _exchangeName = Exchange.BINANCEUS
-  // GeoTrust RSA CA 2018
+  // *.binance.com - validity not after: 17/02/2024, 00:59:59 CET
   readonly _certFingerprint256 =
-    '8C:C3:4E:11:C1:67:04:58:24:AD:E6:1C:49:07:A6:44:0E:DB:2C:43:98:E9:9C:11:2A:85:9D:66:1F:8E:2B:C7'
+    '93:07:DE:DD:AF:3A:78:77:1D:B1:B7:68:3E:9F:18:8E:28:83:AE:A1:77:58:87:D4:5C:F6:F9:C8:71:1A:72:49'
```

Comment:
this is the certificate + validity from binance.com, not binance.us

Reply:
we use the same certificate for them